autogen/main.tf.tmpl

```diff
@@ -117,6 +118,11 @@ resource "google_compute_backend_service" "default" {
     }
   }
 
+  log_config {
+    enable      = true
```

Comment:
I don't think we should *require* the use of `log_config`, can we please make this a dynamic block? We can make the default be enabled though.